src/main/java/com/epam/lab/collections2/consolemenu/MenuOptions.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.lab.collections2.consolemenu;
+
+public enum MenuOptions {
+    OPTION1(1, "Execute method 1"),
+    OPTION2(2, "Execute method 2"),
+    OPTION3(3, "Execute method 3"),
+    OPTION4(4, "Exit")
+    ;
+
+    private Integer menuOption;
+    private String optionDescription;
+
+    private MenuOptions(Integer menuOption, String optionDescription)
+    {
```

Comment:
prefer to place { in the end of declaring method class etc not from new line, it is also an alternative convention but at list it should be consistent everywhere